src/main/java/app/sami/languageWeb/config/JwtService.java

```diff
@@ -0,0 +1,62 @@
+package app.sami.languageWeb.config;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+
+@Service
+public class JwtService {
+
+    private static final String SECRET_KEY = "2D4B6150645367566B597033733676397924423F4528482B4D6251655468576D";
```

Comment:
Let's not commit a secret key, use spring injection mechanisms instead and load the key from your environment variables

Reply:
Keep it like this for the moment, I'll walk you through the `jwkset` method which is far superior